README.md

```diff
@@ -163,6 +163,12 @@ trl sft --model_name_or_path Qwen/Qwen2.5-0.5B \
     --output_dir Qwen2.5-0.5B-SFT
 ```
 
+```bash
+trl sft --model_name_or_path Qwen/Qwen2.5-0.5B \
+    --dataset_name ./dataset.json \
+    --output_dir Qwen2.5-0.5B-SFT
+```
```

Comment:
Let's keep the readme simple

```suggestion
```

Reply:
ok

Reply:
Do you suggest another place to put the official doc in please? For your information, I did it in my blog at:

https://codeberg.org/libertia/modular-AI-doc#3-train-your-own-model-for-bypassing-privacy-copyright-issue-according-to-your-view